lib/feedjira/date_time_utilities/date_time_epoch_parser.rb

```diff
@@ -0,0 +1,13 @@
+# rubocop:disable Style/Documentation
+# rubocop:disable Style/DocumentationMethod
+module Feedjira
+  module DateTimeUtilities
+    class DateTimeEpochParser
+      def self.parse(string)
+        epoch_time = string.to_i
+        return Time.at(epoch_time).to_datetime if epoch_time.to_s == string
+        raise "#{string} is not a valid epoch time"
```

Comment:
I’m following the lead of [`DateTimeLanguageParser`](https://github.com/feedjira/feedjira/blob/master/lib/feedjira/date_time_utilities/date_time_language_parser.rb) and [`DateTimePatternParser`](https://github.com/feedjira/feedjira/blob/master/lib/feedjira/date_time_utilities/date_time_pattern_parser.rb).